src/main/java/com/minecolonies/coremod/colony/buildings/BuildingFarmer.java

```diff
@@ -262,8 +263,9 @@ public void onDestroyed()
                   .notifyBlockUpdate(scarecrowTileEntity.getPos(),
                     getColony().getWorld().getBlockState(scarecrowTileEntity.getPos()),
                     getColony().getWorld().getBlockState(scarecrowTileEntity.getPos()),
-                          BLOCK_UPDATE_FLAG);
-                scarecrowTileEntity.setName(LanguageHandler.format("com.minecolonies.gui.scarecrow.user", LanguageHandler.format("com.minecolonies.gui.scarecrow.user.noone")));
+                    BLOCK_UPDATE_FLAG);
+                scarecrowTileEntity.setName(LanguageHandler.format("com.minecolonies.coremod.gui.scarecrow.user",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "com.minecolonies.coremod.gui.scarecrow.user" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1192)